packages/plugins/content/slate/src/Component/index.js

```diff
@@ -136,11 +131,11 @@ class Slate extends Component {
           onKeyDown={onKeyDown}
           readOnly={Boolean(readOnly)}
           className="ory-plugins-content-slate-container"
-          onBlur={onBlur}
-          schema={schema}
-          state={editorState}
+          onBlur={this.onBlur}
```

Comment:
Any particular reason for moving `onBlur` to `this`? It doesn't seem like we need the classes context for the blur function here?

Reply:
Just best practice for me (obviously everybody has different sort of rules). In the future, the onBlur might need to use the class even-though now it doesn't. Also it tend to be more straight forward in unit testing. But this obviously is mostly matter of preference. No real harm or benefit comes from this.

Reply:
Personal preference is absolute legitimate. The thing is that large PRs are incredibly difficult to review. Non-functional code changes that have to do with personal preference make it tougher to review as nothing functional has changed, just the style of the code. It's also important that the same code-style is kept throughout the project. This is incredibly hard to achieve but it's a bit easier if contributors try to avoid "personal preference" code-changes here and there.

Rule of thumb is, if a method doesn't require the classes context, move it to a constant. It's more likely that you'll remove the need for the class (specifically in React) and end up with a stateless component than potentially needing the context in some function later. My rule is always: "If in doubt, leave it out".